bindings/rabbitmq/rabbitmq.go

```diff
@@ -55,17 +63,21 @@ type RabbitMQ struct {
 	closed     atomic.Bool
 	closeCh    chan struct{}
 	wg         sync.WaitGroup
+
+	// used for reconnect
+	notifyRabbitChannelClose chan *amqp.Error
 }
 
 // Metadata is the rabbitmq config.
 type rabbitMQMetadata struct {
-	Host             string `json:"host"`
-	QueueName        string `json:"queueName"`
-	Exclusive        bool   `json:"exclusive,string"`
-	Durable          bool   `json:"durable,string"`
-	DeleteWhenUnused bool   `json:"deleteWhenUnused,string"`
-	PrefetchCount    int    `json:"prefetchCount"`
-	MaxPriority      *uint8 `json:"maxPriority"` // Priority Queue deactivated if nil
+	Host             string        `json:"host"`
+	QueueName        string        `json:"queueName"`
+	Exclusive        bool          `json:"exclusive,string"`
+	Durable          bool          `json:"durable,string"`
+	DeleteWhenUnused bool          `json:"deleteWhenUnused,string"`
+	PrefetchCount    int           `json:"prefetchCount"`
+	MaxPriority      *uint8        `json:"maxPriority"` // Priority Queue deactivated if nil
+	ReconnectWait    time.Duration `json:"reconnectWait"`
```

Comment:
Because this is parsed as `reconnectWaitSeconds`, please make this property unexported, i.e.:

```suggestion
	reconnectWait    time.Duration
```

Reply:
Agreed with @ItalyPaleAle

Reply:
Done.